docs/releases/3.2.txt

```diff
@@ -191,7 +191,10 @@ Security
 Serialization
 ~~~~~~~~~~~~~
 
-* ...
+* :ref:`JSONL <serialization-formats-jsonl>`, serializer added,
+  now you can :djadmin:`dumpdata` and :djadmin:`dumpdata` with this format,
+  which can be useful for loading large databases because the output is loaded
+  line by line into memory (instead of loading whole data at once.)
```

Comment:
We've got a `dumpdata` twice there. 

```suggestion
* The new :ref:`JSONL <serialization-formats-jsonl>` serializer allows using 
  the JSON Lines format with :djadmin:`dumpdata` and :djadmin:`loaddata`. This
  can be useful for populating large databases because the data is loaded line by
  line into memory, rather than being loaded all at once.
```